plugins/modules/sonic_prefix_lists.py

```diff
@@ -492,6 +494,34 @@
 #                           }
 #                       ],
 """
+RETURN = """
+before:
+  description: The configuration prior to the model invocation.
+  returned: always
+  type: list
+  sample: >
+    The configuration returned will always be in the same format
+     of the parameters above.
+after:
+  description: The resulting configuration model invocation.
+  returned: when changed
+  type: list
+  sample: >
+    The configuration returned will always be in the same format
+     of the parameters above.
+after(generated):
+  description: The generated configuration model invocation.
+  returned: when C(check_mode)
+  type: list
+  sample: >
+    The configuration returned will always be in the same format
+     of the parameters above.
+commands:
+  description: The set of commands pushed to the remote device.
+  returned: always
+  type: list
+  sample: ['command 1', 'command 2', 'command 3']
+"""
```

Comment:
It looks like we should probably also have some description for the "diff" output. The same issue applies to route_maps as well. (This was probably missed for other check/diff PRs. If necessary, we can have a separate PR to add it to those resource modules.)

Reply:
Added sample texts.